client/state/data-layer/wpcom/account-recovery/lookup/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { isString } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+import {
+	ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+} from 'state/action-types';
+
+export const fromApi = data => ( [
+	{
+		email: data.primary_email,
+		sms: data.primary_sms,
+	},
+	{
+		email: data.secondary_email,
+		sms: data.secondary_sms,
+	},
+] );
+
+const validateFromApi = ( { primary_email, primary_sms, secondary_email, secondary_sms } ) => {
+	return isString( primary_email ) &&
+		isString( primary_sms ) &&
+		isString( secondary_email ) &&
+		isString( secondary_sms )
+	;
```

Comment:
let's check on this formatting - it's hanging in ways it shouldn't - we should at least have some parenthesis

we could also consider another approach here based on arrays…

```js
const validateFromApi = ( { primary_email, primary_sms, secondary_email, secondary_sms } ) => [
	primary_email,
	primary_sms,
	secondary_email,
	secondary_sms,
].every( isString );
```